lib/nexthop_group.c

```diff
@@ -232,6 +329,35 @@ uint32_t nexthop_group_hash(const struct nexthop_group *nhg)
 	return key;
 }
 
+uint32_t nexthop_group_hash(const struct nexthop_group *nhg)
+{
+	struct nexthop *nh;
+	uint32_t key = 0;
+
+	for (ALL_NEXTHOPS_PTR(nhg, nh))
+		key = jhash_1word(nexthop_hash(nh), key);
+
+	return key;
+}
+
+void nexthop_group_mark_duplicates(struct nexthop_group *nhg)
```

Comment:
A fair point. They were in the code before so I elected to keep them at least for now to avoid unforeseen errors. But I think we could get rid of them in a future PR to optimize. I don't think it hurts much to leave them in for now.